canonical-web-and-design/ubuntu.com
# [Engage] Delete duplicate of ITstrategen case study page

## Done

Deleted duplicate of https://ubuntu.com/engage/itstrategen-case-study

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/casestudy-itstratagen
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
  - Ensure that the page no longer exists.


## Issue / Card

Fixes #5541
